### PR TITLE
Chore/disk mgt UI fixes

### PR DESCRIPTION
**Pull request by joshenlim**
Add warning if at 90% + within cooldown period
Screenshot 2024-11-07 at 18 11 13

Add warning if in read only mode
Screenshot 2024-11-07 at 18 35 12

Also updated bar to be red if the utilization is >= 90% (since DB will enter read only mode at 95%)

### CODE REVIEW

File: apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx [hunk 1]
```diff
@@ -310,6 +311,35 @@ export function DiskManagementForm() {
           />
           {!isFlyArchitecture && (
             <>
+              <div className="flex flex-col gap-y-3">
+                <DiskCountdownRadial />
+                {usedPercentage >= 90 && isWithinCooldownWindow && (
+                  <Admonition
+                    type="destructive"
+                    title="Database is currently at 90% of disk size"
```
Comment (jordienr) [inline apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx:319]: should be `is currently over 90%` right?

File: apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx [hunk 2]
```diff
@@ -310,6 +311,35 @@ export function DiskManagementForm() {
           />
           {!isFlyArchitecture && (
             <>
+              <div className="flex flex-col gap-y-3">
+                <DiskCountdownRadial />
+                {usedPercentage >= 90 && isWithinCooldownWindow && (
```
Comment (jordienr) [inline apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx:316]: should it have `&& !isReadOnlyMode` too? Or is that checked in `isWithinCooldownWindow`

**Review by jordienr — APPROVED** [review]

File: apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx (hunk 1 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx:319]: oh yeap! good point ahha

File: apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx (hunk 2 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx:316]: ahhh yes great catch thank you so much! 🙏 will need to add this check or 2 warnings will show i think

**Pull request merged**